unittests/AST/ASTImporterTest.cpp

```diff
@@ -1007,7 +1015,8 @@ TEST_P(ASTImporterTestBase, ImportRecordTypeInFunc) {
   ASSERT_TRUE(FromVar);
   auto ToType =
       ImportType(FromVar->getType().getCanonicalType(), FromVar, Lang_C);
-  EXPECT_FALSE(ToType.isNull());
+  llvm::Error Err = ToType.takeError();
+  EXPECT_FALSE(Err.operator bool());
```

Comment:
Do we really have to call `operator bool` explicitly?

Reply:
It is not needed.